.github/workflows/release.yml

```diff
@@ -0,0 +1,105 @@
+name: "Release New Version"
+run-name: "Release ${{ inputs.version }}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The version to be released (e.g. 1.2.3)"
+        required: true
+        type: "string"
+
+env:
+  GH_TOKEN: ${{ github.token }}
+  # TODO: Adding the mongodb-dbx-release-automation app to the repository will allow fetching a one-time token and pushing
```

Comment:
This seems like a necessary thing to enable in the initial version, as we do have branch protection rules enabled.

Reply:
Correct. In that case, you should follow [these instructions (internal link)](https://wiki.corp.mongodb.com/display/SYSENG/GitHub#GitHub-RequestingaGitHubApplication) to request the `mongodb-dbx-release-bot` application to be added to all relevant repositories.

Reply:
Requested

Reply:
🍿 Is this the last blocker?

Reply:
It is - just waiting for our IT to enable the app here, then we're good to go :)